addons/stock/static/src/xml/inventory_lines.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="InventoryLines.Buttons">
+        <button type="button" class='btn btn-primary o_stock-widget-pdf'>
```

Comment:
widget-pdf?

Reply:
Bad copy/paste I think... I don't remember but it seems useless and not declared in CSS.